### PR TITLE
Cover component created

### PR DESCRIPTION
**Pull request by AmanRaj1608**
image

### CODE REVIEW

**Comment by rossoskull** [comment]
Please deploy it to [surge](https://surge.sh/) and comment the link.

**Comment by pAditya198** [comment]
@AmanRaj1608 Please remove package-lock.json file

**Comment by rossoskull** [comment]
image

@AmanRaj1608 
Check the design for date labels, the position and text needs to be changed (Marked by **red**)

Also there will be separators (Marked by **green**)

**Comment by AmanRaj1608** [comment]
https://cerebro-aman.surge.sh/

Having problem in adding animation and making vertical line length bigger.

image

File: src/Components/Home/Cover/Countdown.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  flex-direction: column;
+
+  strong {
+    font-size: 50px;
+  }
+}
+
+.vl {
+  border-left: 2px solid #56ceca;
+  left: 50%;
```
Comment (samagragupta) [inline src/Components/Home/Cover/Countdown.scss]: Not needed.

File: src/Components/Home/Cover/Countdown.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  margin: 0 20px;
+  display: flex;
+  flex-direction: column;
+
+  strong {
+    font-size: 50px;
+  }
+}
+
+.vl {
```
Comment (samagragupta) [inline src/Components/Home/Cover/Countdown.scss:24]: Add `font-size: 3rem;`

File: src/Components/Home/Cover/Countdown.scss (hunk 1 above)
Comment (AmanRaj1608) [reply src/Components/Home/Cover/Countdown.scss]: Done

File: src/Components/Home/Cover/Countdown.scss (hunk 2 above)
Comment (AmanRaj1608) [reply src/Components/Home/Cover/Countdown.scss:24]: Yes that worked. Thanks

File: src/Components/Home/Cover/Countdown.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.Countdown-col {
+  display: inline-block;
+}
+
+.Countdown-col-element {
+  display: inline-block;
+  margin: 0 20px;
+  display: flex;
```
Comment (samagragupta) [inline src/Components/Home/Cover/Countdown.scss:16]: Why two display are given?

Comment (AmanRaj1608) [reply src/Components/Home/Cover/Countdown.scss:16]: Fixed

**Review by samagragupta — APPROVED** [review]

**Pull request merged**